src/App.css

```
@@ -1,54 +1,130 @@
+
+/* --- Dark Mode Overrides --- */
+body.body-dark-mode {
+  background-color: var(--bg-color-dark);
+  color: var(--text-color-dark);
+}
+
+body.body-dark-mode a {
+  color: var(--link-color-dark);
+}
+body.body-dark-mode a:hover {
+  color: lighten(var(--link-color-dark), 10%); /* Slightly lighten on hover */
```

Biome (1.9.4)

[error] Unexpected unknown function: lighten

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)